### DISCUSSION TITLE
Prefix names used as arguments

### DISCUSSION THREAD

**Discussion by soegaard** [racket/rhombus#189] [Q&A]
The following program:
```
#lang rhombus
import:
  "racket-port.rkt":   prefix port

fun read(port): port.to_string(port)
```
results in the error 
```
to_string: no such field
  in value: #{'#<input-port:/Users/soegaard/Dropbox/GitHub/rhombus-experiments/big.txt>}

``` 
This makes sense, since in `port.to_string` port is now bound to the input argument.
The intention was however to use `to_string` which was imported from "racket-port.rkt".

Is there a way to get the `to_string` imported from "racket-port.rkt" if the identifier `port` is shadowed?

The file `racket-port.rkt`.
```
#lang racket/base
(require racket/port)
(provide
 (rename-out [port->string to_string]))

```

**Comment by plane** [comment]
In many languages, there is a way to refer to the global or top-level namespace.  For example, in C++ you would write a leading `::` before an identifier.  It's not clear to me how to do this in Rhombus.

Along those lines, my first thoughts were something like:

```
fun read(port): .port.to_string(port)        // or
fun read(port): ().port.to_string(port)      // or
fun read(port): _.port.to_string(port)
```

Maybe there is a better way to do it.
